src/Support/ResolveInfoFieldsAndArguments.php

```diff
@@ -76,6 +85,7 @@ public function __construct(ResolveInfo $info)
      * @param  int  $depth  How many levels to include in output
      * @return array
      * @see \GraphQL\Type\Definition\ResolveInfo::getFieldSelection
+     * @throws \Exception
```

Comment:
Please remove all the `@throws` you added in this PR

They are deliberately not added because it's a PITA to maintain them. Code changes, exception changes and before you notice it, it's out of date.